modules/watson-machine-learning/main.tf

```diff
@@ -0,0 +1,99 @@
+# create COS instance for WatsonX.AI project
+module "cos" {
+  providers = {
+    ibm = ibm.ibm_resources
+  }
+  source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version           = "8.3.2"
+  resource_group_id = var.resource_group_id
+  cos_instance_name = var.cos_instance_name
+  cos_plan          = "standard"
+}
+
+## Use code from Watson SaaS directly to avoid "legacy module" issues
+## Note: passing a non-null delegated storage attribute may result in API errors
+
+
+resource "restapi_object" "configure_project" {
```

Comment:
Keep note of removing duplicate logic here and use directly the module from watsonx repo.